src/scenarios/mauiandroid/test.py

```diff
@@ -1,11 +1,26 @@
 '''
 C# Console app
 '''
+import os
 from shared.runner import TestTraits, Runner
 
 EXENAME = 'MauiAndroidDefault'
+MAUIVERSIONFILE = 'MAUI_VERSION.txt'
 
 if __name__ == "__main__":
+    try:
+        with open(f'pub/{MAUIVERSIONFILE}', 'r') as f:
+            maui_version = f.read()
+            if("sha" not in maui_version or "azdo" not in maui_version):
+                raise ValueError(f"MAUI_VERSION does not contain sha and azdo indicating failure to retrieve or set the value. MAUI_VERSION: {maui_version}")
+            else:
+                print(f"Found MAUI_VERSION {maui_version}")
+                os.environ["MAUI_VERSION"] = maui_version
+        os.remove(f'pub/{MAUIVERSIONFILE}')
```

Comment:
What's the purpose of removing the file?

Reply:
The reason for the file being removed is to ensure it is not included in any Size On Disk tests. If I find a good workaround to not use a file, then this will not be necessary.